pygatt/backends/gatttool/gatttool.py

```diff
@@ -38,11 +40,126 @@ def wrapper(self, connected_device, *args, **kwargs):
     return wrapper
 
 
+class GATTToolReceiver(threading.Thread):
+    """
+    Observe pygatttool stdout in seperate thread and dispatch events /
+    callbacks.
+    """
+
+    def __init__(self, connection, parent_aliveness):
+        super(GATTToolReceiver, self).__init__()
+        self.daemon = True
+        self._connection = connection
+        self._parent_aliveness = parent_aliveness
+        self._event_vector = {
+            'notification': {
+                'pattern': r'Notification handle = .*? \r',
+            },
+            'indication': {
+                'pattern': r'Indication   handle = .*? \r',
+            },
+            'invalid_fd': {
+                'pattern': r'.*Invalid file descriptor.*',
+            },
+            'disconnected': {
+                'pattern': r'.*Disconnected\r',
+            },
+            'char_written': {
+                'pattern': r'Characteristic value written successfully',
```

Comment:
On my version of gatttool I get the following message:

[C0:F0:84:3C:51:FA][LE]> char-write-req 0x13 0dff0000                                                                                                                                                                                                                                                                        
Characteristic value was written successfully